--- config/FindSUPERLUMT.orig.cmake	2016-09-27 00:16:47.000000000 +0200
+++ config/FindSUPERLUMT.cmake	2016-10-04 12:08:58.202988791 +0200
@@ -60,7 +60,7 @@
 endif()
 
 # Set SuperLU_MT library name with thread type postfix
-set(SUPERLUMT_LIBRARY_NAME superlu_mt_${SUPERLUMT_THREAD_TYPE})
+set(SUPERLUMT_LIBRARY_NAME superlumt64_d)
 
 if(MSVC)
   set(CMAKE_FIND_LIBRARY_PREFIXES lib ${CMAKE_FIND_LIBRARY_PREFIXES})

